schema/deploy/shipping/views.sql

```diff
@@ -3271,7 +3271,7 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         join warehouse.individual using (individual_id)
         join shipping.uw_reopening_enrollment_fhir_encounter_details_v1 using (encounter_id)
         left join uw_individual_summaries on uw_individual_summaries.individual = individual.identifier
-        where encounter.details @> '{"_provenance": {"redcap": {"url": "https://redcap.iths.org/", "project_id":23854, "event_name": "enrollment_arm_1"}}}'
```

Comment:
👍 I think this is right for the uw_priority_queue view. We'd only want to be sending out invitations in the new project once we roll over.